lib/services/events/index.ts

```diff
@@ -53,8 +54,8 @@ export function registerWebhookHandler(type: WebhookEventType, handler: WebhookH
  * @param {WebhookEventBase} type
  * @param {WebhookPayload} payload
  */
-export async function handleWebhookEvent(event: WebhookEventBase) {
-  await each(getHandlersForType(event.type), (h : WebhookHandler) => h(event));
+export async function handleWebhookEvent(event: WebhookEventBase, cfg: GoodChatConfig) {
```

Comment:
Why the dependency injection?

Reply:
```
export async function onConversationCreated(event: WebhookEventBase) : Promise<void> {
// ...
}
```

Shouldn't the above function then take a `config` as its second argument?

Reply:
if it's not using the second argument there's no need to take it. typescript allows that